pkg/kube/cluster-init.sh

```diff
@@ -202,12 +202,12 @@ check_start_containerd() {
                 ln -s /var/lib/rancher/k3s/data/current/bin/containerd-shim-runc-v2 /usr/bin/containerd-shim-runc-v2
         fi
 
-        if pgrep -f "containerd --config" >> $INSTALL_LOG 2>&1; then
-                logmsg "k3s-containerd is alive"
-        else 
-                logmsg "Starting k3s-containerd"
+        pgrep -f "containerd --config" > /dev/null 2>&1
```

Comment:
Shoudn't this be if [ $? -eq 0 ]; then ?

Reply:
If the exit status is 0 it matched at least one pid.  https://man7.org/linux/man-pages/man1/pkill.1.html. I suppose this could be changed to if [ $? -eq 1 ]; then to match on no processes found.  as the other two non-zero codes imply other issues.